app/representers/api/media_resource_representer.rb

```diff
@@ -2,10 +2,14 @@ class Api::MediaResourceRepresenter < Roar::Decorator
   include Roar::JSON
   include HalApi::Representer::FormatKeys
 
-  property :href
-  property :original_url, writeable: false
+  property :href, readable: false
+  property :file_name
   property :mime_type, as: :type
   property :file_size, as: :size
   property :duration
   property :status, writeable: false
+
+  # TODO: deprecate
+  property :href
+  property :original_url, writeable: false
```

Comment:
oh, I expected these to be removed, and only available in he `auth` version?
I guess they will be, but not yet in this PR.

Reply:
Yep, this PR doesn't change the API yet.  But it enables us to deploy https://github.com/PRX/dovetail-router.prx.org/pull/260, moving DTR to use all auth endpoints.

After that DTR change, we can come back and deprecate the href here.